src/StorageSync/StorageSync/StorageSyncService/NewStorageSyncServiceCommand.cs

```diff
@@ -110,8 +110,8 @@ public class NewStorageSyncServiceCommand : StorageSyncClientCmdletBase
             Mandatory = false,
             HelpMessage = HelpMessages.StorageSyncServiceIdentityTypeParameter)]
         [ValidateSet(StorageSyncServiceIdentityType.systemAssigned,
-            StorageSyncServiceIdentityType.userAssigned,
-            StorageSyncServiceIdentityType.systemAssignedUserAssigned,
+            //StorageSyncServiceIdentityType.userAssigned,
```

Comment:
this is breaking change removing values from validate set

Reply:
Not really
- we never supported this value from service